examples/kubernetes/secret.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mongo-express
+data:
+  db-server-url: "bW9uZ29kYi5kZWZhdWx0LnN2Yy5jbHVzdGVyLmxvY2FsCg==" # replace with you mongodb url
```

Comment:
I think it would be better if you put a comment to tell people this is a base64 and which value is this decoded

Reply:
you could put a readme.md in kubernets folder instead of comment in the file

Reply:
I added the readme with a short description on how it is setup and linking to the kubernetes documentation for the secret.